index.html

```diff
@@ -69,142 +74,150 @@ <h2 class="yoda">
+            </ul>
+            <button class="seeproject">See Project</button>
+          </div>
+          <img src="Snapshoot Portfolio(3).svg" alt="" class="gridpicture4" />
         </div>
       </div>
     </section>
     <section class="about">
       <div class="about-container">
-        <h2 class="yodaa">About Myself</h2>
-        <p class="intro-text">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website. Look through some of my work and experience! If
-          you like what you see and have a project you need coded, don’t
-          hesitate to contact me.
-        </p>
-        <p class="connect">LET'S CONNECT</p>
-
-        <ul class="social-media">
-          <li class="social-icon">
-            <a href="">
-              <img src="images/github.svg" alt="github" />
-            </a>
-          </li>
-          <li class="social-icon">
-            <a href="">
-              <img src="images/Linkedin.svg" alt="Linkedin" />
-            </a>
-          </li>
-          <li class="social-icon">
-            <a href="">
-              <img src="images/victory.svg" alt="victory" />
-            </a>
-          </li>
-          <li class="social-icon">
-            <a href="#">
-              <img src="images/twitter.svg" alt="twitter" />
-            </a>
-          </li>
-          <li class="social-icon">
-            <a href="">
-              <img src="images/microverse.svg" alt="Meta" />
-            </a>
-          </li>
-        </ul>
+        <div class="about-container-text">
+          <h2 class="yodaa">About Myself</h2>
+          <p class="intro-text2">
+            Hello I’m a software developer! I can help you build a
+            product , feature or website. Look through some of my 
+            work and experience! If you like what you see and have
+            a project you need coded, don’t hesitate to contact me.
+          </p>
+          <p class="connect">LET'S CONNECT</p>
 
-        <p class="resume">Get my resume</p>
+          <ul class="social-media">
+            <li class="social-icon">
+              <a href="">
+                <img src="images/github.svg" alt="github" />
+              </a>
+            </li>
+            <li class="social-icon">
+              <a href="">
+                <img src="images/Linkedin.svg" alt="Linkedin" />
+              </a>
+            </li>
+            <li class="social-icon">
+              <a href="">
+                <img src="images/victory.svg" alt="victory" />
+              </a>
+            </li>
+            <li class="social-icon">
+              <a href="#">
+                <img src="images/twitter.svg" alt="twitter" />
+              </a>
+            </li>
+            <li class="social-icon">
+              <a href="">
+                <img src="images/microverse.svg" alt="Meta" />
+              </a>
+            </li>
+          </ul>
 
+          <p class="resume">Get my resume</p>
```

Comment:
> [OPTIONAL] As this was not pointed out by the previous code reviewer, kindly use a button tag instead of a p tag. This uses the semantics tag correctly and is friendly for the SEO😊🚀